### ISSUE TITLE
morph tools

### ISSUE THREAD

**Issue by Kenji233iy** [MMD-Blender/blender_mmd_tools#184]
## What version of softwares are you using?
 - Blender: 4.2.3 LTS
 - mmd_tools:  4.2.2
 - OS: Win 10H

when i want to edit the facial expressions with the shape keys, they dont even appear, so i go to the mmd_tools menu under morph tools and it is shown that my expressions shape keys are in the "bones" tab not in the "vertex" tab so i cant edit or even use them, any idea of i can fix it?
![Captura de pantalla 2024-11-20 002756](https://github.com/user-attachments/assets/a0ad61a8-6b4d-404b-9aec-b9946bf586c7)

**Comment by rintrint** [comment]
Yes, you can. You can edit it. In your picture, the bone offsets setting is located in the bottom-left corner. That’s how bone morphing works. Blender only supports vertex morphs natively. The other four types of morphs are specific to MMD. The mmd_tools add-on uses drivers to make them compatible. Therefore, before using them, you need to click the assemble morph button. These are not shape keys—only vertex morphs are considered shape keys.
<img width="280" alt="Snipaste_2024-11-21_16-42-17" src="https://github.com/user-attachments/assets/8ddbace3-06b2-43b3-9b78-832a92c6e4ad">
